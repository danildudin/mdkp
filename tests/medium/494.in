29 5 0
893 721 11 100 673 171 152 198 461 952 429 149 893 891 709 438 791 663 478 209 91 51 535 463 869 345 184 180 595 
376 103 431 785 733 734 226 913 39 386 739 13 89 939 622 458 904 821 888 985 851 151 357 649 915 185 66 896 456 
858 865 418 271 743 277 719 639 980 379 119 798 739 667 137 317 821 338 990 886 776 194 396 650 237 158 348 123 566 
53 731 438 311 959 771 615 223 273 1 356 50 473 55 628 498 838 610 387 576 377 554 551 71 300 965 637 538 793 
269 493 244 334 365 362 8 637 714 773 201 322 167 733 82 29 838 175 683 640 536 132 160 886 380 537 20 287 54 
470 194 599 930 6 901 702 649 864 737 576 18 887 717 324 894 986 496 419 86 78 415 888 923 965 720 441 784 63 
5633 9219 6463 6091 6443 
