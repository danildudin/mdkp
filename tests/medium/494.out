8653
13
0 1 4 8 9 11 12 13 14 17 24 26 28 
